third_party/build/bin/ryu_build.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+#
+# Copyright 2022 The Magma Authors.
+
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# package ryu with magma patches
+
+set -e
+SCRIPT_DIR="$(dirname "$(realpath "$0")")"
+# shellcheck source=/dev/null # relative path unknown and option -x not given
+source "$SCRIPT_DIR"/../lib/util.sh
+PKGVERSION=4.34
+ITERATION=1
+# shellcheck disable=SC2034 # variable is used in third_party/build/lib/util.sh
+VERSION="$PKGVERSION-$ITERATION"
+PKGNAME=python3-ryu
```

Comment:
Is it too late or is there another reason not to call this package `python3-ryu-magma`?

Reply:
the name is created via `build.py` and `utils.sh`.

Reply:
as above